tsconfig.json

```diff
@@ -23,5 +23,8 @@
     "scripts",
     "src",
     "stories"
+  ],
+  "exclude": [
+    "stories/setup/webpack.config.js"
```

Comment:
It's a bit weird that this triggers an error but it does:

<img width="608" alt="screenshot 2019-02-04 11 49 00" src="https://user-images.githubusercontent.com/55398/52226615-14fd3780-2873-11e9-9dcf-1fa7f4aab527.png">